src/_listDefinitions.scss

```diff
@@ -40,6 +40,22 @@ $fd-list-navigation-navigator-bar-right: inset -0.1875rem 0 0 0 var(--sapSelecte
 $fd-list-navigation-navigator-bar-left: inset 0.1875rem 0 0 0 var(--sapSelectedColor);
 $fd-list-navigation-indicator-width: 2.75rem !default;
 
+$fd-list-object-border-bottom: 1px solid var(--sapList_BorderColor);
+$fd-list-object-with-value-zero: 0;
+$fd-list-object-with-padding-value-half: 0.5rem;
+$fd-list-object-with-padding-value-one: 1rem;
+$fd-list-object-with-padding-value-three: 3rem;
+$fd-list-object-with-value-half: 0.75rem;
+$fd-list-object-align-left: left;
+$fd-list-object-align-right: right;
+$fd-list-object-with-vertical-align-top: top;
+$fd-list-object-with-vertical-align-bottom: bottom;
+$fd-list-object-with-no-text-shadow: 0 0 black;
```

Comment:
Is this correct? I can't find it in the spec. If it should be here, it should definitely be using a variable. It looks pretty strange:
With:
![Screen Shot 2020-09-25 at 10 01 09 AM](https://user-images.githubusercontent.com/29607818/94295492-2b5b7980-ff16-11ea-968e-31611de191d1.png)
![Screen Shot 2020-09-25 at 10 01 17 AM](https://user-images.githubusercontent.com/29607818/94295536-3adac280-ff16-11ea-937e-f9d6a1486edf.png)

Reply:
Object variants such as status, number by default have a shadow effect on text when it is active but in the case of object lists item, these shadow effects should not be present as per the images shared in Fiori guidelines. To stay in sync with object list item styles look and feel this property is updated here.

As per Fiori.

<img width="518" alt="2020-09-26_00-30-00" src="https://user-images.githubusercontent.com/53487468/94305831-7b2a5700-ff8f-11ea-91cc-e66479ef233b.png">



Without text-shadow update look in local.

<img width="388" alt="2020-09-26_00-19-22" src="https://user-images.githubusercontent.com/53487468/94305903-97c68f00-ff8f-11ea-993c-ee14f944d0e5.png">

Reply:
Can you show me where this is defined in the design spec? It's not mentioned in ObjectList or ObjectNumber:
![Screen Shot 2020-09-25 at 12 43 26 PM](https://user-images.githubusercontent.com/29607818/94309360-ceb78900-ff2c-11ea-9a50-38de2ae861ec.png)
![Screen Shot 2020-09-25 at 12 43 52 PM](https://user-images.githubusercontent.com/29607818/94309362-d1b27980-ff2c-11ea-90af-3696679d7b39.png)

Reply:
Yes, that's right, inspite of this the current styles of object number and object status are showing shadow effect on text which in turn effecting object list item look as mentioned in the above images.

Reply:
Sorry, I don't think I'm following. Why are you adding text shadow? Are you doing that based on the image you added with a blue background? If so, where is that coming from?

Reply:
Object number and object status.scss has defined text-shadow (though it is not mentioned is the spec) which is resulting in updated text-shadow to match as per-object list item look as mentioned in the screenshot above.

Reply:
I'm sorry, I'm still not following. How does it have a defined text-shadow if it's not mentioned in the spec? Are there incorrect styles in `object-status.scss` that should be removed instead of doing hacks to override them?

Reply:
Doing some research, I see now where it's added. That contrast in objectStatus was added for accessibility reasons - https://github.com/SAP/fundamental-styles/pull/1571. We shouldn't need to override it here.

Reply:
As per my understanding from the spec, yes we should not have text-shadow in object number and object status styles. 

As am not ware of the requirement behind adding the text-shadow there, I prefer to override it in object list item (As a temporary fix).

Until it is fixed (if it is considered as bug) in object number and object status.

Reply:
The current text shadow was added as an accessibility fix. @jacobdevera can you provide additional context for the text shadow added in  https://github.com/SAP/fundamental-styles/pull/1571?

A possible override (if @jacobdevera can confirm that his changes were meant only for when the components are in a table and not in general), would be to set `text-shadow: none`.

Reply:
The `text-shadow` was added primarily for Table when rows are selected as the selected color causes contrast issues. If it needs to be overridden, it makes sense to use `text-shadow: none`, as the black `text-shadow` currently being set seems off.

The reason it looks off in your screenshot is due to the inverted styling (white text against dark background) making it appear to have a glow effect. I also set the `text-shadow` to `none` in those cases as there doesn't seem to be an appropriate `text-shadow` variable to use for inverted text.

Reply:
That said, the Object List does appear to have a selection mode with semantic text, which would run into the same problem as Table as it uses the same colors as Table. If it must be added to the spec to be incorporated here, the text-shadow should be added or be accepted as a known deviation, as the Positive text does _narrowly_ go below a 4.5:1 contrast without the text shadow.